### DISCUSSION TITLE
Feature request - 'tree' mode - ability to select text

### DISCUSSION THREAD

**Discussion by talgershman** [josdejong/svelte-jsoneditor#440] [Ideas]
Hello,

Currenlty in 'tree' mode when clicking a string value, all the cell is highlight ( the cursor is 'pointer') and the user can't mark the text ( only if he press edit then he can edit the cell).

I would like an ability highlight only part of the text ( without the need to edit the cell before ).

for example: 
https://jsonformatter.org/json-editor/55f614
here I can hightlight/mark when my cursor only part of the url and copy it.

**Comment by josdejong** [comment]
You're referring to the "classic" version of the JSON Editor: https://github.com/josdejong/jsoneditor.

The successor of this editor, `svelte-jsoneditor` has a different way of working (similar to cells in Excel). Many things are easier (and finally possible), like copy/paste of a value in your JSON document. But some things have become a bit harder, like you first have to double click to start editing a value.

Right now, to select a part of a value, you can double click the value to edit it, and then select the part you want to copy.

It would be nice indeed to be able to partially select a value directly. It may be possible, I'm not sure. Anyone interested in trying to make that work?

Moving this idea to the Ideas section now...
